release-content/0.15/release-notes/15538_Add_Support_for_Triggering_Events_via_AnimationEvents.md

```diff
@@ -1,4 +1,34 @@
 <!-- Add Support for Triggering Events via `AnimationEvent`s -->
 <!-- https://github.com/bevyengine/bevy/pull/15538 -->
 
-<!-- TODO -->
+Sometimes it is useful for animations to drive game logic. In **Bevy 0.15** we've added "animation event" support to [`AnimationClip`], which means that you can trigger a specific [`Event`] at a given point in time during [`AnimationClip`] playback:
```

Comment:
```suggestion
Triggering gameplay events when animations start or finish is a common pattern for synchronizing the visual and mechanical parts of your game. In **Bevy 0.15** we've added "animation event" support to [`AnimationClip`], which means that you can trigger a specific [`Event`] at a given point in time during [`AnimationClip`] playback:
```

Reply:
"Start or finish" is too specific / doesn't cover the full scope. It would be better to say "at a given point in time". But unfortunately I've already said that, so it feels repetitive.